format/FlightSql.proto

```diff
@@ -867,6 +867,185 @@ enum SqlSupportsConvert {
   SQL_CONVERT_VARCHAR = 19;
 }
 
+/**
+ * It is an attribute that identifies the type of any object. It's
+ * used by JDBC and ODBC to expose the values defined by the DBMC.
+ */
+enum XDBCDataType {
+  XDBC_TYPE_UNKNOWN_TYPE = 0;
+  XDBC_TYPE_CHAR = 1;
+  XDBC_TYPE_NUMERIC = 2;
+  XDBC_TYPE_DECIMAL = 3;
+  XDBC_TYPE_INTEGER = 4;
+  XDBC_TYPE_SMALLINT = 5;
+  XDBC_TYPE_FLOAT = 6;
+  XDBC_TYPE_REAL = 7;
+  XDBC_TYPE_DOUBLE = 8;
+  XDBC_TYPE_DATETIME = 9;
+  XDBC_TYPE_INTERVAL = 10;
+  XDBC_TYPE_VARCHAR = 12;
+  XDBC_LONGVARCHAR = -1;
+  XDBC_BINARY = -2;
+  XDBC_VARBINARY = -3;
+  XDBC_LONGVARBINARY = -4;
+  XDBC_BIGINT = -5;
+  XDBC_TINYINT = -6;
+  XDBC_BIT = -7;
+  XDBC_WCHAR = -8;
+  XDBC_WVARCHAR = -9;
```

Comment:
I'm curious: is there a reason some types have negative values? Also, the naming is a bit inconsistent (`XDBC_TYPE_something` vs. `XDBC_something`).

Reply:
Fixed

Reply:
same question about negative values?

Reply:
I think in all cases where enums correspond to existing standard numeric values it should be noted in the docs.

Reply:
I've added a doc to say that